operator/src/main/java/org/keycloak/operator/controllers/KeycloakDeploymentDependentResource.java

```diff
@@ -296,6 +298,7 @@ private StatefulSet createBaseDeployment(Keycloak keycloakCR, Context<Keycloak>
                         && (customImage.isPresent() || operatorConfig.keycloak().startOptimized())) {
             containerBuilder.addToArgs(OPTIMIZED_ARG);
         }
+        containerBuilder.addToArgs(0, "-Djgroups.bind.address=$(%s)".formatted(POD_IP));
```

Comment:
Nitpick: Add a comment stating why we do this.

```suggestion
        // Set bind address as this is required for JGroups to form a cluster in IPv6 envionments
        containerBuilder.addToArgs(0, "-Djgroups.bind.address=$(%s)".formatted(POD_IP));
```